research-software-directory/RSD-as-a-service
# 83 commits chart

# Commits line chart

Closes #83

Changes proposed in this pull request:

*  Fix empty repositoryInfo for new software (not scraped)
* Add line chart to software page
* Add percentage text to programming languages

How to test:

* `docker-compose down --volumes`: to remove old data. Not sure if needed
* `docker-compose build`: build the solution
* run data-migration script
* open app on localhost and visit some of the software pages
* Confirm you see line chart with commits and the progamming languages. Note! the commit stats differ from legacy RSD. I believe this is due to github metrics api which seem to explude PR / merge commits. **Note! If you just removed all data, the scrapers will need some time to scrape all the data again.**

![image](https://user-images.githubusercontent.com/9204081/154855773-a185dba2-0bb8-4f3f-b574-e293c8007fb2.png)

PR Checklist:
